scripts/sync-and-test-bucket.sh

```diff
@@ -49,6 +49,11 @@ aws_region="$(pulumi -C infrastructure config get 'aws:region')"
 # fail, causing this script to exit nonzero. In either case, it's okay to continue.
 aws s3 mb $destination_bucket_uri --region $aws_region || true
 
+# Tag the bucket with ownership information for production buckets.
+if [ "$(pulumi stack --show-name)" == "production" ]; then
```

Comment:
Looking at the log, I think this should be:

```suggestion
if [ "$(pulumi -C infrastructure stack --show-name)" == "production" ]; then
```